bin/install.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eu
+
+# Ensure PHP is installed
+if ! command -v php > /dev/null 2>&1; then
+  echo "PHP is not found, installation aborted!"
+  exit 1
+fi
+
+# Ensure Composer is installed/updated
+if ! command -v composer > /dev/null 2>&1; then
+  echo "*** Installing Composer ***"
+  php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+
+  if [ \
+      "$(curl -S https://composer.github.io/installer.sig)" = \
+      "$(php -r \"echo hash_file\('SHA384', 'composer-setup.php'\);\")" \
+  ]; then
+      php composer-setup.php --quiet --filename=composer
+      sudo mv composer /usr/local/bin/composer
```

Comment:
Not entirly sure about the `sudo`, then again, `chown`ing would probably also require sudo, so... meh.